content/src/quick-action-tasks/architecture-license-reinstatement.md

```diff
@@ -6,34 +6,32 @@ urlSlug: architecture-license-reinstatement
 callToActionText: Reinstate My Architecture License
 callToActionLink: https://www.njconsumeraffairs.gov/arch/applications/Application-for-Reinstatement-with-Instructions.pdf
 ---
+
 You need to submit a reinstatement application if:
 
-*  Your architecture license has been expired for five years or less
+- Your architecture license has been expired for five years or less
 
-:::infoAlert 
- If your license has been expired for over five years, then you need to re-apply for your architect license.
+:::infoAlert
+If your license has been expired for over five years, then you need to re-apply for your architect license.
 :::
 
```

Comment:
I assume that quick actions are like licenses and allowed to have no SummaryMD?

Reply:
That's correct!